app/models.py

```diff
@@ -1843,3 +1843,15 @@ def serialize(self):
             'complaint_date': self.complaint_date.strftime(DATETIME_FORMAT) if self.complaint_date else None,
             'created_at': self.created_at.strftime(DATETIME_FORMAT),
         }
+
+
+class ServiceDataRetention(db.Model):
+    __tablename__ = 'service_data_retention'
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), unique=False, index=True, nullable=False)
+    service = db.relationship(Service, backref=db.backref('service_data_retention'))
+    notification_type = db.Column(notification_types, nullable=False)
+    days_of_retention = db.Column(db.Integer, nullable=False)
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    updated_at = db.Column(db.DateTime, nullable=True, default=datetime.datetime.utcnow)
```

Comment:
Should the `updated_at` column be `onupdate=datetime.datetime.utcnow` instead of `default=datetime.datetime.utcnow` or do they do the same thing?

Reply:
It doesn't change the column on the table, but it will populate the value on update if not provided, so it's a good idea to change it to onupdate